spk/transmission/Makefile

```diff
@@ -13,7 +13,7 @@ DESCRIPTION_SPN = Transmission es un cliente BitTorrent simple y rápido. Puedes
 ADMIN_PORT = 9091
 RELOAD_UI = yes
 DISPLAY_NAME = Transmission
-CHANGELOG = "Enable sc-download group"
+CHANGELOG = "Update Transmission to 2.92"
```

Comment:
Let's include the previous changelog as well: in this particular case, there were only a very select number of arches that received a package with it included (alpine and newer, or something like that):
```
CHANGELOG = "1. Update Transmission to 2.92<br>2. Enable sc-download group"
```